worker-workflow/src/test/resources/input-document-no-subdoc-with-stack.json

```diff
@@ -3,7 +3,7 @@
         "reference": "ref_1_file",
         "fields": {
             "CAF_WORKFLOW_ACTION": [{
-                    "data": "super_action"
+                    "data": "family_hashing"
```

Comment:
Why are all these test actions being renamed?  What has that got to do with this Jira?

Reply:
Since I have now modified the workflow-control script to only check for failures if the worker has `terminateOnFailure == true`, I needed to pass a workflow action that was coherent with it. I defined 3 of them in the  `createInvocableNashornEngine()` method, when I want to process failures, I use the family_hashing action, when I want to simualte a `terminateOnFailure == true`, I use the bulk_indexer.